docs/rules.md

```diff
@@ -126,6 +126,7 @@ Rules are sorted by severity.
 | `KEY_FIREFOX_ANDROID_UNSUPPORTED_BY_MIN_VERSION`        | warning  | Manifest key not compatible with Firefox for Android at `applications.gecko.strict_min_version` |
 | `PERMISSION_FIREFOX_UNSUPPORTED_BY_MIN_VERSION`         | notice   | Permission not compatible with `applications.gecko.strict_min_version`                          |
 | `PERMISSION_FIREFOX_ANDROID_UNSUPPORTED_BY_MIN_VERSION` | notice   | Permission not compatible with Firefox for Android at `applications.gecko.strict_min_version`   |
+| `RESTRICTED_HOMEPAGE_URL`                               | error    | Linking to AMO domain name in `homepage_url` is not allowed                                     |
```

Comment:
Nit, let's replace AMO with addons.mozilla.org as in the message description.

Reply:
There is a conflict with the changes in this file, I guess your git branch has been branched from an older version of the main addons-linter branch, and so this PR needs to resolve that conflict (by either rebase your branch, or merging the last version of the main branch into yours).

Let me know if you needs help with that, I'll be happy to provide some additional guidance about how to do it.

Reply:
@rpl  Sure i'll try to fix the conflict, i made the other changes.
I'll let you if i need help with fixing the conflict.
Thanks.

Reply:
@rpl Done. please let me know if i missed anything.